lib/matplotlib/axes/_axes.py

```diff
@@ -677,6 +677,14 @@ def axhline(self, y=0, xmin=0, xmax=1, **kwargs):
         -------
         :class:`~matplotlib.lines.Line2D`
 
+        Other parameters
```

Comment:
Turns out there were quite a few cases of this already, I've changed them all for consistency to an uppercase P.